front/src/modules/trainschedule/components/DriverTrainSchedule/DriverTrainScheduleStop.tsx

```diff
@@ -1,114 +0,0 @@
-import { useMemo } from 'react';
-
-import cx from 'classnames';
-
-import type { PositionSpeedTime, SpeedRanges } from 'reducers/simulationResults/types';
-
-import type { OperationalPointWithTimeAndSpeed } from './types';
-import { getActualSpeed, getActualVmax, getAverageSpeed, getTime } from './utils';
```

Comment:
`getActualSpeed`, `getAverageSpeed` and `getTime` can be removed (and maybe some functions using them)

Reply:
Done